pyeudiw/satosa/backend.py

```diff
@@ -424,10 +429,19 @@ def request_endpoint(self, context, *args):
 
         # TODO: take the response and extract from jwt the public key of holder
         
-        entity_id = self.db_engine.init_session(
-            context.http_headers['HTTP_DPOP'], 
-            context.http_headers['HTTP_AUTHORIZATION']
-        )
+        try:
+            entity_id = self.db_engine.init_session(
+                dpop_proof=context.http_headers['HTTP_DPOP'], 
+                attestation=context.http_headers['HTTP_AUTHORIZATION']
+            )
+        except Exception as e:
+            return JsonResponse(
```

Comment:
please add the logging call here

Reply:
Resolved in [c69e7db](https://github.com/italia/eudi-wallet-it-python/pull/89/commits/c69e7db13eddb6b3a16fc2e5d182fea4b08cfba8)